helm/cluster-api-provider-azure/templates/admissionregistration.k8s.io_v1_validatingwebhookconfiguration_capz-validating-webhook-configuration.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if eq .Values.provider.flavor "capi" }}
```

Comment:
I still see the filter defined in the `objectSelector` , for example line 54 and 79 

```
  objectSelector:
    matchLabels:
      cluster.x-k8s.io/watch-filter: capi
```

should we just remove it from the `kustomize` step in hack/generate-kustomize-patches.sh ?